src/Core_Command.php

```diff
@@ -158,6 +161,12 @@ public function download( $args, $assoc_args ) {
 			$download_url = str_replace( '.zip', '.tar.gz', $offer['download'] );
 		}
 
+		if ( true === \WP_CLI\Utils\get_flag_value( $assoc_args, 'skip-content' ) ) {
+			$offer = unserialize( self::_read( 'https://api.wordpress.org/core/version-check/1.6/' ) );
```

Comment:
Any reason for not using `https://api.wordpress.org/core/version-check/1.7/` which provides JSON data?

Reply:
Hey @ocean90, I opted for /1.6/ as it's used here https://github.com/AaronRutley/core-command/blob/650fad28ce28d5b38c598a2c0654bceac0f562d1/src/Core_Command.php#L272 

I'm happy to swap it for /1.7/  if needed!

Reply:
Gotcha, maybe there's a reason... cc @danielbachhuber

Reply:
Here's the original commit: https://github.com/wp-cli/wp-cli/commit/3fbef8ba71eb62a8e37b8b5d874e1cc351c57c6f

My assumption is that 1.7 wasn't available.

Reply:
Thanks @danielbachhuber, 
Would you like to continue using 1.6 or should we change to 1.7?